mixin-utils/test/test_remove_rules.libsonnet

```diff
@@ -0,0 +1,284 @@
+local utils = import '../utils.libsonnet';
+local test = import 'github.com/jsonnet-libs/testonnet/main.libsonnet';
+
+local config = {
+  prometheusAlerts: {
+    groups: [
+      {
+        name: 'group1',
+        rules: [
+          {
+            alert: 'alert1',
+          },
+          {
+            alert: 'alert2',
+          },
+        ],
+      },
+      {
+        name: 'group2',
+        rules: [
+          {
+            alert: 'alert3',
+          },
+          {
+            alert: 'alert4',
+          },
+        ],
+      },
+    ],
+  },
+  prometheusRules: {
+    groups: [
+      {
+        name: 'group3',
+        rules: [
+          {
+            record: 'record1',
+          },
+          {
+            record: 'record2',
+          },
+        ],
+      },
+      {
+        name: 'group4',
+        rules: [
+          {
+            record: 'record3',
+          },
+          {
+            record: 'record4',
+          },
+        ],
+      },
+    ],
+  },
+};
+
+test.new(std.thisFile)
+
++ test.case.new(
+  'removeAlertRuleGroup',
+  test.expect.eq(
+    actual=config + utils.removeAlertRuleGroup('group1'),
```

Comment:
Thank you for adding tests. This also shows how these functions are supposed to be used! That's very useful!